osf/metadata/utils.py

```diff
@@ -0,0 +1,86 @@
+from website import settings
+
+
+SUBJECT_SCHEME = 'bepress Digital Commons Three-Tiered Taxonomy'
+
+RESOURCE_TYPE_MAP = {
+    'Audio/Video': 'Audiovisual',
+    'Dataset': 'Dataset',
+    'Image': 'Image',
+    'Model': 'Model',
+    'Software': 'Software',
+    'Book': 'Text',
+    'Funding Submission': 'Text',
+    'Journal Article': 'Text',
+    'Lesson': 'Text',
+    'Poster': 'Text',
+    'Preprint': 'Text',
+    'Presentation': 'Text',
+    'Research Tool': 'Text',
+    'Thesis': 'Text',
+    'Other': 'Text',
+    '(:unas)': 'Other'
+}
+
+
+def datacite_format_contributors(contributors):
+    """
+    contributors_list: list of OSFUsers to format
```

Comment:
Minor: I'm not sure what the exact docstring syntax is, but I think it's something like: 
```
:param contributors list: OSFUsers
:returns list: Contributors serialized according to the Datacite schema
```